### PR TITLE
[FR] Mapping Techniques

### PR DESCRIPTION
**Pull request by Slyyme**
French translation of osu! wiki - Mapping Techniques

### CODE REVIEW

**Review by DoryanR — CHANGES_REQUESTED** [review]
After modifications, it will be good for me.

**Comment by DoryanR** [comment]
@Slyyme Need you to apply the proposed changes

File: wiki/Mapping_Techniques/fr.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# Techniques de Mapping
+
+Les techniques de Mapping sont des méthodes de placements/patterns qui sont largement utilisés par les beatmappers connus dans leurs maps.
```
Comment (DoryanR) [inline wiki/Mapping_Techniques/fr.md:3]:
```suggestion
Les **techniques de mapping** sont des méthodes de placements/patterns qui sont largement utilisés par les mappers connus dans leurs beatmaps.
```

File: wiki/Mapping_Techniques/fr.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
+# Techniques de Mapping
```
Comment (DoryanR) [inline wiki/Mapping_Techniques/fr.md:1]:
```suggestion
# Techniques de mapping
```

File: wiki/Mapping_Techniques/fr.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+Les techniques de Mapping sont des méthodes de placements/patterns qui sont largement utilisés par les beatmappers connus dans leurs maps.
+
+Ces pages sont ici comme guides pour débutant, pour aider à expliquer comment faire de telles techniques jusqu'à ce que vous ayez développé votre propre style de mapping. Pour commencer, il est recommandé d'observer et de copier ces techniques de mapping à partir de mappers expérimentés tout en permettant à votre créativité de circuler librement petit-à-petit. Beaucoup de grands mappers ont aiguisé leurs compétences à leurs débuts en imitant les autres qu'ils admiraient, il n'y a donc aucune honte à ce que vous le faites vous-même.
```
Comment (DoryanR) [inline wiki/Mapping_Techniques/fr.md:5]:
```suggestion
Ces pages sont un guide pour les débutant, pour leurs expliquer comment réaliser de telles techniques jusqu'à ce que vous ayez développé votre propre style de mapping. Pour commencer, il est recommandé d'observer et de copier ces techniques de mapping à partir de mappers expérimentés tout en permettant à votre créativité de circuler librement petit-à-petit. Beaucoup de grands mappers ont aiguisé leurs compétences à leurs débuts en imitant les autres qu'ils admiraient, il n'y a donc aucune honte à ce que vous le faites vous-même.
```

File: wiki/Mapping_Techniques/fr.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+Vous trouverez ci-dessous des liens vers certaines techniques de mapping connues, et toutes largements utilisées par les mappers.
```
Comment (DoryanR) [inline wiki/Mapping_Techniques/fr.md:7]:
```suggestion
Vous trouverez ci-dessous des liens vers certaines techniques de mapping connues, et toutes largement utilisées par les mappers.
```

File: wiki/Mapping_Techniques/fr.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+Si nécessaire, vous pourriez également obtenir un essentiel des techniques de mapping à [Techniques de Mapping/Notions de base](Basics/).
```
Comment (DoryanR) [inline wiki/Mapping_Techniques/fr.md:8]:
```suggestion
Si nécessaire, vous pourriez également obtenir un essentiel des techniques de mapping sur l'article [techniques de mapping/Notions de base](Basics/).
```

File: wiki/Mapping_Techniques/fr.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+- [Square Formation](Formations/#square-formation)
+- [Square Formation v2](Formations/#square-formation-v2)
+- [Triangle Beats](Formations/#triangle-beats)
+- [Zig-Zag Sliders](Formations/#zig-zag-sliders)
+
+### Jumps
+
+*Main page: [Mapping Techniques/Jumps](Jumps)*
+
+#### Hit Circle
```
Comment (DoryanR) [inline wiki/Mapping_Techniques/fr.md:47]:
```suggestion
#### Cercles
```

**Review by DoryanR — CHANGES_REQUESTED** [review]

**Comment by DoryanR** [comment]
@Slyyme Are you still on the job?

File: wiki/Mapping_Techniques/fr.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+- [Slider Stream Jumps](Rhythm/#slider-stream-jumps)
+- [Triplet Streams](Rhythm/#triplet-streams)
+- [Variable Spaced Streams](Rhythm/#variable-spaced-streams)
+- [Zig-Zag Streams](Rhythm/#zig-zag-streams)
+
+### Spinners
+
+*Main page: [Mapping Techniques/Spinners](Spinners)*
+
+#### Effets Additionnels
```
Comment (DoryanR) [inline wiki/Mapping_Techniques/fr.md:96]:
```suggestion
#### Effets additionnels 
```

File: wiki/Mapping_Techniques/fr.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+### Spinners
+
+*Main page: [Mapping Techniques/Spinners](Spinners)*
+
+#### Effets Additionnels 
+
+- [Post-Spinner Note](Spinners#post-spinner-note)
+- [Pre-Spinner Note](Spinners#pre-spinner-note)
+
+#### Variaions de Spinners
```
Comment (DoryanR) [inline wiki/Mapping_Techniques/fr.md:101]:
```suggestion
#### Variations de spinners
```

**Review by DoryanR — CHANGES_REQUESTED** [review]

**Comment by DoryanR** [comment]
@TicClick I think we can apply the suggestions and merge, because it's been too long since the person gave up their PR.

**Review by TicClick — APPROVED** [review]

**Pull request merged**